pandas/core/arrays/sparse.py

```diff
@@ -27,7 +27,7 @@
     pandas_dtype)
 from pandas.core.dtypes.dtypes import register_extension_dtype
 from pandas.core.dtypes.generic import (
-    ABCIndexClass, ABCSeries, ABCSparseSeries)
+    ABCIndexClass, ABCSeries, ABCSparseArray, ABCSparseSeries)
```

Comment:
I think you want to use this where you currently have forward declarations too?

Reply:
Ran mypy against the changed files without the blacklist, and there aren't any new problems.